src/tikzplotly/_data.py

```diff
@@ -31,6 +32,14 @@ def data_type(data):
         return None
     return None
 
+def treat_data(data_str): 
```

Comment:
I think that when I merge origin into this, I accidentally remove your modifications of `data_type` and `post_treat_data`, sorry about that... Could you please check ?
At this point, the function `treat_data` is used nowhere, so I must have done something wrong :/